### DISCUSSION TITLE
Does the newly discovered model api support contextual conversations？

### DISCUSSION THREAD

**Discussion by zjrwtx** [waylaidwanderer/node-chatgpt-api#59] [Help] [answered]

**Comment by waylaidwanderer** [comment, marked as answer]
As per the readme, my library handles the conversation context in a nearly identical way to ChatGPT

**Comment by zjrwtx** [comment]
> As per the readme, my library handles the conversation context in a nearly identical way to ChatGPT

all right,thanks

**Comment by JerryWn12** [comment]
sorry a noob question, how to stay in one conversation? do i need just post the coinversationId the server provided ?
such as:

send:
```
{
    "message": "how are you?"
}
```
receive:
```
{
   "response": "...",
   "conversationId": "abc",
   ...
}
```

second send:
```
{
    "message": "what is my previous question?",
    "conversaionId": "abc"
}
```

can i recevice my first question? Thanks!

**Comment by waylaidwanderer** [comment]
It should be explained in the readme. Check it out and let me know if you have any questions.

**Comment by JerryWn12** [comment]
> It should be explained in the readme. Check it out and let me know if you have any questions.

thank you, i passed server the parentMessageId and it worked
